### PR TITLE
Add "example integrations" section.

### PR DESCRIPTION
**Pull request by vext01**
For now demonstrates how to use pizauth with msmtp and mbsync. More can be added later if need be.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,3 +182,41 @@ Where:
...
+mail utilities to query pizauth when an authentication token is required. This
+section contains example configuration snippets to help you get up and running.
+
+In these examples, text in chevrons (like `<this>`) needs to be edited to match
+your individual setup. We also assume that `pizauth` is in `$PATH`, but if it
+is not, most tools allow you to use absolute paths instead.
+
+### msmtp
+
+In `$XDG_CONFIG_HOME/msmtp/config` ( or `~/.msmtprc`, whichever you use):
```
Comment (ltratt) [inline README.md:199]: For simplicities sake, let's just say `In ~/.config/msmtp/config:` (people using weird/old versions will be able to adjust to their filename).

Comment (vext01) [reply README.md:199]: Well, I still use `~/.msmtprc`. It's still a valid config path in the latest msmtp.

Comment (ltratt) [reply README.md:199]: Yes, but you will hardly struggle to translate "~/.config/msmtp/config" to "~/.msmtprc". And most people don't know what XDG_CONFIG_HOME is. Let's just use the recommended modern setting in its simple form, and us all crusties can adjust as necessary :)

Comment (vext01) [reply README.md:199]: `XDG_CONFIG_HOME` isn't necessarily `~/.config` so hard-coding that is wrong too.

Comment (vext01) [reply README.md]: Compromise on [7ba2c96](https://github.com/ltratt/pizauth/pull/1/commits/7ba2c9629f0debe3694f773d73e550aeb695f30e)?

Comment (ltratt) [reply README.md]: Works for me! [And I know about XDG_CONFIG_HOME, which pizauth honours, but am also aware that few people really know what it is.]

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Comment by ltratt** [comment]
Thanks!

**Pull request merged**